### DISCUSSION TITLE
Ilyama ProLite XUB2793QSU-B6

### DISCUSSION THREAD

**Discussion by Dutchiiee** [xanderfrangos/twinkle-tray#814] [Monitor Compatibility Issues]
Hi There,
I tried changing settings with this monitor through ControlMyMonitor and this works perfectly but it seems like Twinkle Tray refuses to work.
Could you please advice?

Thanks in advance! :)

**Comment by Dutchiiee** [comment]
To quickly add to this, i managed to make it work by reinstalling Twinkle Tray and using the Microsoft Store version instead of using the download on the website.

**Comment by xanderfrangos** [comment]
@Dutchiiee Do you know if you were using the beta version previously?

**Reply by Dutchiiee** [reply]
Hi Xander,

I just checked the version i downloaded and it seems to be version 1.15.5.
When i checked though i currently have the same version installed..

Perhaps what it could be is that first i couldn't change the settings on the monitor either?
Due to using an scene in the Ilyama Prolite my brightness controls etc. where greyed out.

After tinkering with the settings and then reinstalling the application might have fixed it?

EDIT: Maybe the DDC/DI feature only get polled on start up of the application?

Hope this helps!
